test/verify/check-accounts

```diff
@@ -112,6 +112,7 @@ class TestAccounts(MachineCase):
         b.set_checked(admin_role_sel, True)
         b.wait_present(admin_role_sel + ":checked")
         b.wait(lambda: "jussi" in m.execute("grep %s /etc/group" % m.get_admin_group()))
+        b.wait_present("#account-roles .alert-info")
```

Comment:
I think this needs to be guarded to not get checked on rhel-7-5. Unfortunately TestAccounts seems to have gotten broken recently on rhel-7-5 (already fails earlier on), which might hide this issue.

Reply:
Fixed.